File: app/src/main/java/com/callmangement/EHR/models/FeedbackbyDCListRoot.java
package com.callmangement.EHR.models;

import java.util.ArrayList;

public class FeedbackbyDCListRoot {
    public ArrayList<FeedbackbyDCListDatum> Data;
    public String message;
    public String status;


    public FeedbackbyDCListRoot(ArrayList<FeedbackbyDCListDatum> data, String message, String status) {
        Data = data;
        this.message = message;
        this.status = status;
    }


    public ArrayList<FeedbackbyDCListDatum> getData() {
        return Data;
    }

    public void setData(ArrayList<FeedbackbyDCListDatum> data) {
        Data = data;
    }

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }

    public String getStatus() {
        return status;
    }

    public void setStatus(String status) {
        this.status = status;
    }
}

File: app/src/main/java/com/callmangement/ehr/models/FeedbackbyDCListRoot.kt
package com.callmangement.ehr.models

import com.google.gson.annotations.SerializedName

class FeedbackbyDCListRoot(
    @SerializedName("Data")
    var data: ArrayList<FeedbackbyDCListDatum>,
    @JvmField var message: String,
    @JvmField var status: String
)
